kratos/solving_strategies/schemes/residual_implicit_time_scheme.h

```diff
@@ -0,0 +1,530 @@
+//    |  /           |
```

Comment:
what is the purpose of this file?

Reply:
The purpose is to reduce code duplication among implicit schemes with this